src/zino/api/server.py

```diff
@@ -21,18 +21,31 @@ class ZinoServer:
     API_PORT = 8001
     NOTIFY_PORT = 8002
 
-    def __init__(self, loop: AbstractEventLoop, state: ZinoState, polldevs: dict[str, PollDevice]):
+    def __init__(
+        self, loop: AbstractEventLoop, state: ZinoState, polldevs: dict[str, PollDevice], config: Configuration = None
```

Comment:
```suggestion
        self, loop: AbstractEventLoop, state: ZinoState, polldevs: dict[str, PollDevice], config: Optional[Configuration] = None
```

Reply:
Committed this directly from GitHub, but now the ruff formatter is not happy :stuck_out_tongue: